common/src/main/java/com/datastrato/gravitino/utils/MapUtils.java

```diff
@@ -10,6 +10,8 @@
 import java.util.Map;
 
 public class MapUtils {
+  private MapUtils() {}
+  
```

Comment:
Hi @MohitKambli I think you have some style issues here. Please see the CI result here https://github.com/datastrato/gravitino/actions/runs/7758988885/job/21176380140?pr=1976.

I guess you have **whitespaces** in this blank. You need to remove these **whitespaces** with just a blank line.

<img width="793" alt="image" src="https://github.com/datastrato/gravitino/assets/850797/4de57669-2c27-4fa4-9700-74e50f5ee77a">

Reply:
@MohitKambli 
Can you add a file name `.gitattributes` with the content `* text eof=lf` and try again, this issue may be caused by the value of line breaks in different OS systems, thanks.

Reply:
![Uploading image.png…]()